x-pack/packages/kbn-elastic-assistant/impl/connectorland/connector_setup/index.test.tsx

```diff
@@ -65,11 +65,11 @@ describe('useConnectorSetup', () => {
       ).toEqual([
         {
           username: 'You',
-          timestamp: 'at: 7/17/2023, 1:00:36 PM',
+          timestamp: `at: ${new Date('2024-03-18T18:59:18.174Z').toLocaleString()}`,
```

Comment:
nit: The test is implemented in the way obscuring testing data. Some dates are specified as expected here but it comes from the other file `../../mock/conversation`. If it's changed by someone it breaks the test and most probably not only this one. Having a concrete input data and expected output will make the test easier to understand and less fragile. On top of that it's possible to split the test into testing comments without timestamps and testing timestamps separately.